internal/templates/default/image-based-installer/template.go

```diff
@@ -13,7 +13,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 */
 
-package imagebasedinstall
+package imagebasedinstaller
```

Comment:
I feel like the default directory isn't necessary.

It implies that there will be some directory for non-default templates which I don't think there ever will be.

I think `engine`, `image-based-installer`, and `assisted-installer` are fine directories under `templates`.

Reply:
Agreed! Changed.